package/MDAnalysis/lib/util.py

```diff
@@ -1274,3 +1275,23 @@ def blocks_of(a, n, m):
                    a.strides[0], a.strides[1])
 
     return np.lib.stride_tricks.as_strided(a, new_shape, new_strides)
+
+
+class Namespace(object):
```

Comment:
Yes, this was something I was going to come back to, probably setting up requirements for a 'valid name' - having a `__getitem__` works quite nicely though, so I might go with that, thanks.